src/World.h

```diff
@@ -79,7 +79,7 @@ typedef cItemCallback<cFlowerPotEntity>    cFlowerPotCallback;
 
 
 
-
+const int CHUNK_CHANCE_ACCURACY = 100000;
```

Comment:
Why so high? Also, have you considered unloading the chunks that are the oldest unused? I guess that's even a valid reason to introduce a new variable...